File: prime.py
def is_prime(num):
    """ Return True is the number is prime. """
    if num < 2:
        return False
    for n in range(2, num):
        if num % n == 0:
            return False
    return True

def primes_lessthan(number):
    """ Return a list of all prime numbers less than n. """
    result = []
    for i in range(2, number):
        if is_prime(i):
            result.append(i)
    return result

print(primes_lessthan(50))

File: Prime.java
import java.util.ArrayList;
import java.util.List;
import java.util.Arrays;

public class Prime {

	private static boolean isPrime(int num) {
		// Return true if the number is prime
		if (num < 2) {
			return false;
		}
		for (int i = 2; i < num; i++) {
			if (num % i == 0) {
				return false;
			}
		}
		return true;
	}

	private static void primesLessThan(int num) {
		// Print a list of all primes less than num.
		List<Integer> primes = new ArrayList<>();
		for (int i = 2; i < num; i++) {
			if (isPrime(i)) primes.add(i);
		}
		System.out.println(Arrays.toString(primes.toArray()));
	}

	public static void main(String [] args) {
		primesLessThan(50);
	}
}
